packages/amplify-provider-awscloudformation/test/s3.test.js

```diff
@@ -0,0 +1,28 @@
+const S3 = require('../src/aws-utils/aws-s3.js');
```

Comment:
If this is part of the unit test, then it should mock all AWS related calls. If you want to use actual AWS resources, then this should be moved to amplify-e2e-tests. When running the tests on PR, we don't have aws creds and this test will always fail